Update Phoenix doc (1.5.6) and add LiveView

sabiwara opened the pull request:
If you're updating an existing documentation to it's latest version, please ensure that you have:

- [x] Updated the versions and releases in the scraper file
- [x] Ensured the license is up-to-date and that the documentation's entry in the array in `about_tmpl.coffee` matches it's data in `self.attribution`
- [x] Ensured the icons and the `SOURCE` file in <code>public/icons/*your_scraper_name*/</code> are up-to-date if the documentation has a custom icon
- [x] Ensured `self.links` contains up-to-date urls if `self.links` is defined
- [x] Tested the changes locally to ensure:
  - The scraper still works without errors
  - The scraped documentation still looks consistent with the rest of DevDocs
  - The categorization of entries is still good

![Screenshot from 2020-08-21 08-08-26](https://user-images.githubusercontent.com/11598866/90834604-a1077f00-e385-11ea-9de8-abbcac64af77.png)

simon04 reviewed (changes requested):
I spotted two issues – the homepage is broken, the headings are broken:
![2020-11-14-204345](https://user-images.githubusercontent.com/782446/99155653-1dc2a480-26ba-11eb-8b3f-f9bcb2ab1d5d.png)
![2020-11-14-204313](https://user-images.githubusercontent.com/782446/99155654-1ef3d180-26ba-11eb-81ad-b31656b8f5e2.png)

sabiwara commented:
Thanks for the review, and your work on this project :heart: @simon04 

I rebased on master, it fixed the issue. I also updated from 1.5.4 to 1.5.6 since it is now the latest version.

![Screenshot from 2020-11-15 20-00-24](https://user-images.githubusercontent.com/11598866/99183207-727b2380-277d-11eb-9827-203f7c02f0d8.png)
![Screenshot from 2020-11-15 20-01-20](https://user-images.githubusercontent.com/11598866/99183202-6f803300-277d-11eb-85db-2d90d4c8c5c8.png)

Side note: there are some remaining issues (also in the master branch) that I didn't manage to fix:
```
  https://hexdocs.pm/ecto/                                          
  Docs::ContainerFilter::ContainerNotFound: element '#content' could not be found in the document, url=https://hexdocs.pm/ecto/
```

simon04 reviewed (approved):
Thank you!
